### PR TITLE
Add .travis.yml

### PR DESCRIPTION
**Pull request by dkukharau**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+services:
+- docker
+
+language: generic
+
+jobs:
+  include:
+    # Latest image build/push
+    - if:  tag IS blank OR commit_message =~ /^Triggered by upstream build/
```
Comment (Evgeniy-L) [inline .travis.yml]: Can you give more details about the `commit_message` here. Does it comes from user? Why do we handle `Triggered by upstream build` differently?

Comment (dkukharau) [reply .travis.yml]: This message is used when gentool travis job is triggered by on of the plugin repositories.

Comment (Evgeniy-L) [reply .travis.yml]: If I add the same message to my commit will it trigger a job?

Comment (dkukharau) [reply .travis.yml]: Yes, you will.

Comment (dkukharau) [reply .travis.yml]: It will trigger it anyway regardless of the message, because it's expected to run this job for testing PRs/commits.

Comment (dkukharau) [reply .travis.yml]: It's just to make sure versioned images are not built by plugin-triggered jobs.

**Pull request merged**